modules/tauri/index.ts

```diff
@@ -9,8 +9,10 @@ export default defineNuxtModule({
     const nuxt = useNuxt()
     const { resolve } = createResolver(import.meta.url)
 
-    if (!process.env.TAURI_PLATFORM)
+    if (!process.env.TAURI_PLATFORM) {
+      addImports({ name: 'useNativeSettings', from: resolve('./runtime/native-settings-undefined') })
```

Comment:
Could you please explain why it is needed to have this `-undefined` version?
Cant this be e.g. inlined like `addImports({ name: 'useNativeSettings', from: () => undefined })` or similar?

Reply:
I didn't find any way to add an import inline, the import object requires `from: string` value, and I couldn't find any other auto import releated function, that would do such a thing.